Packs/Tanium/Integrations/Tanium_v2/Tanium_v2.py

```diff
@@ -1054,6 +1054,39 @@ def delete_group(client, data_args):
     return human_readable, outputs, raw_response
 
 
+def get_action_result(client, data_args):
+    actions_ids = argToList(data_args.get('id'))
+    action_res_outputs: List = []
+    action_res_hr: List = []
+
+    for action_id in actions_ids:
+        endpoint_url = '/result_data/action/' + str(action_id)
+        raw_response = client.do_request('GET', endpoint_url)
+        try:
+            devices_raw = raw_response['data']['result_sets'][0]['rows']
+            device_results = []
+            for device in devices_raw:
+                formatted_device = {
+                    'HostName': device['data'][0][0]['text'],
+                    'Status': str(device['data'][1][0]['text']).split(':')[1],
+                    'ComputerID': device['cid']
+                }
+                device_results.append(formatted_device)
+            raw_response = raw_response.get('data')
+            raw_response['ID'] = action_id
+            action_res_outputs.append(raw_response)
+            action_res_hr.extend(device_results) if device_results else None
+        except Exception:
```

Comment:
The `except` should have the same indentation as the `try`

Reply:
They are.. 
![image](https://user-images.githubusercontent.com/57212002/150128128-b63e618c-8579-4ff5-9b9c-0d25a7e895a0.png)